app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, src, dst = command.split()
```

Comment:
1. You must validate the command. Your code will not work with this
command = "mv file1.txt"
command = "mv file1.txt file2.txt file3.txt"
2. Do not use abbreviations use full names. src = source_file_path the same with dst
3. Why you use _ for first? If i say command = "not_mv file1.txt file2.txt" the file will be moved. 
https://github.com/mate-academy/py-move-file/blob/master/checklist.md
Check checklist